### ISSUE TITLE
Craft 4 Install fail

### ISSUE THREAD

**Issue by CreateSean** [hillholliday/Craft-User-Manual#31]
I saw that user manual is now ready for Craft 4 so I attempted to install it however after running the composer command from the readme and also tried from the Craft plugin store

```composer require roberskine/craft-user-manual```

```composer require roberskine/craft-user-manual && php craft plugin/install usermanual```

It fails with this error message in red:

```
  [InvalidArgumentException]                                                 
  Could not find a matching version of package roberskine/craft-user-manua   
  l. Check the package spelling, your version constraint and that the pack   
  age is available in a stability which matches your minimum-stability (be   
  ta).  
```

**Comment by RobErskine** [comment]
@CreateSean you work fast!

Yeah, I just updated the README with the correct composer information. 

Try:

```composer require hillholliday/craft-user-manual```

The issue as was [outlined in this issue](https://github.com/RobErskine/Craft-User-Manual/issues/26) is that a package can not be renamed after it has been submitted to Packagist. Since I took this on from my old org I thought it could be changed but needs to go back to what it was originally registered at.

Let me know if that works for you and I can close this ticket.
